src/sagemaker/estimator.py

```diff
@@ -152,8 +152,46 @@ def fit(self, inputs, wait=True, logs=True, job_name=None):
         self.latest_training_job = _TrainingJob.start_new(self, inputs)
         if wait:
             self.latest_training_job.wait(logs=logs)
+
+    @classmethod
+    def _from_training_job(cls, init_params, hyperparameters, image, sagemaker_session):
```

Comment:
Add documentation here, it's important. You're introducing a protocol that subclasses need to follow, so this should be documented.